templates/base/src/reactQuery/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import { useQuery, useQueryClient } from 'react-query'
+
+
+const fetchListFromAPI = async () => {
+    const res = await fetch('https://swapi.dev/api/people/');
+    return res.json();
+}
+
+export default function ReactQuerySample({ children }) {
```

Comment:
There is no point of this example here, these kind of examples will be created in components & Documentation. 

From here you need to export the QueryProvider so that it can be applied to application in next.js and plain react.